api/src/main/java/org/apache/apex/api/users/ControlTuple.java

```diff
@@ -16,15 +16,15 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-package org.apache.apex.api;
+package org.apache.apex.api.users;
```

Comment:
We are yet to define these, but I think we can have something like `*ControlTuple`; `EOFControlTuple` or `StartBatchControlTuple` for example. 
Perhaps we can use operator in the package name (`org.apache.apex.api.operator`) which also signifies that these are generated by the operator...